### DISCUSSION TITLE
What exactly means that "deta.sh is retiring"?

### DISCUSSION THREAD

**Discussion by begoon** [deta/cloud-docs#758] [General] [answered]
Does it mean that the existing "deta.sh" deployments at the end of March 2023 will disappear or stop functioning?

Does "retiring" mean that the deta.sh-based projects MUST be migrated manually to Space?

I'm asking to understand what to do precisely. I have a bunch of deployments in deta.sh. Despite my love to Space (which is, by the way, freaking amazing! kudos to the Deta team), I would not spend time migrating the existing projects to Space if they can just stay running in deta.sh.

**Comment by abdelhai** [comment, marked as answer]
it means the following:
- the ability to create new projects and micros has been disabled since yesterday
- you still can access your old deta.sh projects etc on deta.space (check your canvas)
- you can deploy code, make changes and save data to your micros & bases on deta.sh until the sunset date which is **May 1st 2023**
- **on May 1st 2023**, everything on deta.sh will be deleted and can't be accessed anymore

if it's terribly difficult to migrate your cloud micros to space apps, please let us know at team at deta.space
hope that helped
